### PR TITLE
added routes

### PR DESCRIPTION
**Pull request by mikesch803**
1.convert HTML to JSX (There is no functionality in any of the pages)
2.routes added to pages 

# Files to check
 
- [ ] react-fit-store/src/App.js

## index file
- [ ] react-fit-store/src/assests/images/index.js
- [ ] react-fit-store/src/components/index.js
- [ ] react-fit-store/src/pages/index.js
- [ ] react-fit-store/src/index.js
## icon file
 - [ ] react-fit-store/src/icons/icons.js
## component file
 - [ ] react-fit-store/src/components/aside/Aside.jsx
 - [ ] react-fit-store/src/components/footer/Footer.jsx
 - [ ] react-fit-store/src/components/header/Header.jsx
## page file
- [ ] react-fit-store/src/pages/cart-page/Cart.jsx
- [ ] react-fit-store/src/pages/landing-page/LandingPage.jsx
- [ ] react-fit-store/src/pages/login-page/Login.jsx
- [ ] react-fit-store/src/pages/product-page/Product.jsx
- [ ] react-fit-store/src/pages/signup-page/Signup.jsx
- [ ] react-fit-store/src/pages/wishlist-page/Wishlist.jsx
# files not to be check
**All CSS files, BIN file, Package file**

### CODE REVIEW

File: src/assests/images/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import landingPoster from "../images/landing-poster.png";
+import product1 from "../images/pro1.png";
+import product2 from "../images/pro2.png";
+import tracksuit from "../images/tracksuit.png";
+
+export { landingPoster, product1, product2, tracksuit };
```
Comment (b0llu) [inline src/assests/images/index.js]: you dont need to write inport and export differently. just write export { landingPoster } from "../images/landing-poster.png"; ( export naming will differ with the kind of export you have)

File: src/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { LandingPage } from "./landing-page/LandingPage";
+import { Login } from "./login-page/Login";
+import { Signup } from "./signup-page/Signup";
+import { Cart } from "./cart-page/Cart";
+import { Wishlist } from "./wishlist-page/Wishlist";
+import {Product } from './product-page/Product';
+
+export {LandingPage, Login, Signup,Cart,Wishlist,Product}
```
Comment (b0llu) [inline src/pages/index.js]: same here try for export directly

**Review by b0llu — APPROVED** [review]

File: src/assests/images/index.js (hunk 1 above)
Comment (mikesch803) [reply src/assests/images/index.js]: sure i will change

File: src/pages/index.js (hunk 2 above)
Comment (mikesch803) [reply src/pages/index.js]: thanks, for point out these

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Code is OK as per you convert HTML to JSX but have multiple repetitive code

**Comment by mikesch803** [comment]
> Code is OK as per you convert HTML to JSX but have multiple repetitive code

can you point out please exactly where all these?

**Pull request merged**